cloud_functions/token_refresh/utils/logger_utils.py

```
@@ -0,0 +1,37 @@
+    logger = logging.getLogger(name)
+    logger.setLevel(level)
+
+    # Prevent adding duplicate handlers if this function is called multiple times
+    if logger.hasHandlers():
+        return logger
+
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(level)
+
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)